### PR TITLE
Public Sans 2.001 (Maintenance update 1)

### LINKED ISSUE #322: Public Sans - Bug: Fix G4 tracking code
### Describe the bug

Below is the G4 tracking code that needs to be placed on the [Public sans website](https://public-sans.digital.gov/). Place the code right AFTER the HTML <head> element. (use [this resource for more info](https://datadrivenu.com/where-to-place-google-analytics-tracking-code/))

<script async src="https://www.googletagmanager.com/gtag/js?id=G-HBYXWFP794"></script>
<script>
window.dataLayer = window.dataLayer || [];
function gtag(){dataLayer.push(arguments);}
gtag('js', new Date());
gtag('config', '
gtag('config', 'G-HBYXWFP794');
</script>

### Steps to reproduce the bug

Currently the above mentioned code is not on the site as far as I can tell. Furthermore, the code that IS on the site is not in the correct place.

1. Go to public sans
2. View page source
3. Note that the only google code on the page is at the very bottom. And it's not G4 tracking code.
<html>
<body>

--
  | <script async src="https://www.googletagmanager.com/gtag/js?id=UA-47271808-2"></script>
  | <script>
  | window.dataLayer = window.dataLayer \|\| [];
  | function gtag(){dataLayer.push(arguments);}
  | gtag('js', new Date());
  |  
  | gtag('config', 'UA-47271808-2');

</body>
</html>

### Expected Behavior

The code should be placed right after the <head> element for proper tracking. 

### Related code

_No response_

### Screenshots

_No response_

### System setup

_No response_

### Additional context

_No response_

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/public-sans/issues) for duplicate bug reports.

### PR DESCRIPTION
**Pull request by thisisdano**
Closes #322.

## General

- Adds issue and PR templates to the repo
- Includes a number of dependency updates
- This release does not affect any of the font files and makes no change to the distribution ZIP

### CODE REVIEW

**Review by mejiaj — APPROVED** [review]

**Review by mejiaj — APPROVED** [review]
No issues found with build.

File: FONTLOG.txt [hunk 1]
```diff
@@ -14,6 +14,12 @@ To contribute to Public Sans please read [CONTRIBUTING.md](https://github.com/us
 
 See [the Public Sans releases page](https://github.com/uswds/public-sans/releases) for more details about changes to the typeface.
 
+March 12 2024 Public Sans v2.001 (Maintenance update 1)
```
Comment (thisisdano) [inline FONTLOG.txt:17]:
```suggestion
December 30 2024 Public Sans v2.001 (Maintenance update 1)
```

File: site/_sass/styles.scss [hunk 2]
```diff
@@ -18,8 +18,9 @@
 @forward "usa-skipnav";
 @forward "usa-media-block";
 @forward "usa-prose";
+@forward "usa-prose";
```
Comment (mejiaj) [inline site/_sass/styles.scss:21]: Looks like we have a duplicate forward of `usa-prose` on line 20.

```suggestion
```

**Review by mejiaj — COMMENTED** [review]
Minor issue, but otherwise LGTM

File: site/_sass/styles.scss (hunk 2 above)
Comment (thisisdano) [reply site/_sass/styles.scss:21]: I'm going top leave this as-is to avoid any more changes in this release, but we'll add it to the next one

**Review by mejiaj — APPROVED** [review]
Approved. Bug found wasn't a part of the changes. Captured in #334.

File: site/_sass/styles.scss (hunk 2 above)
Comment (mejiaj) [reply site/_sass/styles.scss:21]: Sounds good, captured in #334.

**Pull request merged**